beesdoo_worker_status/models/res_config_settings.py

```diff
@@ -0,0 +1,11 @@
+from odoo import fields, models
+
+
+class ResConfigSettings(models.TransientModel):
+    _inherit = "res.config.settings"
+
+    irregular_penalty = fields.Boolean(
+        string="Penalty for irregular with negative counter",
```

Comment:
Penalty for irregular worker with negative counter

Reply:
fixed